public/static/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -0,0 +1,78 @@
+# Setup Google Drive remote
+
+To setup Google Drive remote you should
+[create and configure Google Project](#create-and-configure-google-project) if
+you don't have one yet.
+
+## Create and Configure Google Project
+
+Since Google Drive has tight API usage quotas, creation and configuration of
+your own `Google Project` is required:
+
+1.  Log into the [Google Cloud Platform](https://console.developers.google.com)
+    account.
+2.  Create `New Project` or select available one.
+3.  Click `ENABLE APIS AND SERVICES` and search for `drive` to enable
+    `Google Drive API` from search results.
+4.  Navigate to
+    [All Credentials](https://console.developers.google.com/apis/credentials)
```

Comment:
should it be `Manage credentials`?

Reply:
Finally, have renamed to `APIs & Services`->`Credentials` along with added screenshot.